netcore/Makefile

```diff
@@ -47,6 +51,8 @@ TEST_ASSETS_URL = $(FEED_BASE_URL)/corefx-tests/$(NETCORETESTS_VERSION)/$(TESTS_
 BLAME = $(shell git blame ../configure.ac | grep AC_INIT | cut -f1 -d' ')
 VERSTUB = .$(shell git rev-list --count $(BLAME)..HEAD)
 
+all: runtime bcl
```

Comment:
What is going to trigger this line? If we're expecting it to run because it's the first target, can we move it higher in the file or comment something to that effect. Otherwise a comment describing what uses the "all" entrypoint would be helpful for future navigators.